backend/molgenis-emx2-graphql/src/main/java/org/molgenis/emx2/json/Column.java

```diff
@@ -16,7 +16,7 @@ public class Column {
   private boolean drop = false; // needed in case of migrations
   private String oldName;
   private Integer key = 0;
-  private Boolean required = false;
```

Comment:
this seems to be part of a feature , not a format check, same for other java changes